util.sh

```diff
@@ -52,5 +52,18 @@ FULL_IMAGE_NAME="${REPO}/${OCI_NAME}:${OCI_VERSION}"
 
 export FULL_IMAGE_NAME_ARCH="$FULL_IMAGE_NAME-${ARCH_TO_IMAGE_ARCH[$CPU_ARCH]}"
 
-FCOS_BASE_IMAGE="quay.io/fedora/fedora-coreos:stable"
+# For released images we want the stable stream but for early testing in CI let's use the next stream.
+FOCS_STREAM="${FOCS_STREAM:-stable}"
```

Comment:
was `FOCS` intentional? I guess it's ok as long as it's consistent but somewhere down the line if someone uses `FCOS` elsewhere and it doesn't work, it might take a while to realize.

Reply:
certainly not, silly typo and then just copy/paste without thinking about it. I fix that.